source/deployment/sso-saml-okta.rst

```diff
@@ -44,30 +42,38 @@ If you’d like to use a Mattermost logo for the application, you can download o
 
-	.. image:: ../../source/images/okta_6_attribute_statements.PNG
+	.. image:: ../../source/images/okta_6_attribute_statements.png
 
 10. Select **Next**. Then, set Okta support parameters for the application. Recommended settings:
  - **I’m an Okta customer adding an internal app**
  - **This is an internal app that we have created**
 
 	.. image:: ../../source/images/okta_7_support_configuration.PNG
 
-11. Select **Finish**. On the next screen, select the **Sign On** tab, then select **View Setup Instructions**.
+11. Select **Finish**. 
+
+12. In the Mattermost System Console, go to **Authentication > SAML 2.0**, then set **Override SAML bind data with AD/LDAP information** to **false** if currently set to **true**. You can re-enable `this configuration setting <https://docs.mattermost.com/administration/prev-config-settings.html#override-saml-bind-data-with-ad-ldap-information>`__ later when once setup is complete.
 
-12. Select the **Identity Provider metadata** link, then copy the link from the browser URL box. This will be used during the SAML configuration steps in the next section. 
+13. On the next screen, select the **Sign On** tab, then select **View Setup Instructions**.
+
+14. Select the **Identity Provider metadata** link, then copy the link from the browser URL field. This will be used during the SAML configuration steps in the next section. 
 
 	.. image:: ../../source/images/okta_8_view_instructions.PNG
 
-13. Take note of **Identity Provider Single Sign-On URL** (also known as **SAML SSO URL**), and the Identity Provider Issuer, as both may be needed to configure SAML for Mattermost. Additionally, ensure you download the X.509 Certificate file and save it. You may need to upload it to Mattermost in a later step.
+15. Take note of **Identity Provider Single Sign-On URL** (also known as **SAML SSO URL**), and the Identity Provider Issuer, as both may be needed to configure SAML for Mattermost. 
+
+16. Download the X.509 Certificate file and save it. You may need to upload it to Mattermost in a later step.
 
 	.. image:: ../../source/images/okta_9_view_instructions.PNG
 
 Configure SAML Sign-in for Mattermost
 --------------------------------------
 
-Start the Mattermost server and sign into Mattermost as a System Administrator. Go to **System Console > Authentication > SAML**, and paste the copied Identity Provider Metadata URL in the **Identity Provider Metadata URL** field, then select **Get SAML Metadata from IdP**.
+Start the Mattermost server and sign into Mattermost as a System Administrator. Go to **System Console > Authentication > SAML 2.0**, then paste the copied Identity Provider Metadata URL in the **Identity Provider Metadata URL** field, and select **Get SAML Metadata from IdP**.
+
+        .. image:: ../../source/images/saml-identity-provider-metadata-url.PNG
```

Comment:
Has this image been uploaded? It's not showing in preview but that's probably not an issue if it's a new image and part of this PR.

Reply:
This appears to have been a pre-existing image file that's missing. It's not a new file added as part this PR work.

Reply:
I've confirmed that this image is also missing on Production.

Reply:
@justinegeffen / @cwarnermm The image is currently broken on production https://docs.mattermost.com/deployment/sso-saml-okta.html#configure-saml-sign-in-for-mattermost

Reply:
Thanks, @amyblais. Fixing this with https://github.com/mattermost/docs/pull/4530